planner/core/common_plans.go

```
@@ -1142,6 +1148,21 @@ func (e *Explain) getOperatorInfo(p Plan, id string) (string, string, string) {
 	return estRows, accessObject, operatorInfo
 }
 
+func (e *Explain) prepareJSONInfo(p PhysicalPlan) {
+	currentNode := &JSONOperatorRow{}
+	if err := e.explainPlanInJSONFormat(currentNode, p, "root", "", false); err != nil {
+		return
```

Should we return this error to users instead of ignoring it?